src/graphql/admin/schema.graphql

```diff
@@ -197,6 +199,10 @@ type SettlementViaIntraLedger {
 
 type SettlementViaLn {
   paymentSecret: LnPaymentSecret
+    @deprecated(
+      reason: "This value/type is mismatched with our domain and will be shifted across to the 'preImage' property."
```

Comment:
I don't know if we should be talking about our domain here as from the POV of an API user it is an implementation detail.
A better way to say it might be: `reason: "Improving granularity of the paymentSecret type"` or something like that.